pages/mobile/home.py

```diff
@@ -22,6 +27,29 @@ def __init__(self, base_url, selenium):
     def header_text(self):
         return self.selenium.find_element(*self._header_locator).text
 
+    def is_user_logged_in(self):
+        self.open_menu()
+        return self.selenium.find_element(*self._sign_out_menulocator)
+
+    def is_user_logged_out(self):
+        self.open_menu()
+        return self.selenium.find_element(*self._sign_in_menu_locator)
```

Comment:
We should just check that `is_user_logged_in` is False rather than having two separate methods for each outcome.